lib/graphiti/scoping/paginate.rb

```diff
@@ -1,13 +1,16 @@
 module Graphiti
   class Scoping::Paginate < Scoping::Base
     DEFAULT_PAGE_SIZE = 20
+    VALID_QUERY_PARAMS = [:number, :size, :before, :after]
```

Comment:
QQ: I don't see `:before` implemented here thus far, correct? Is adding that query param for a later stage of this PR?

Reply:
Yeah, I have some `before` code locally but removed it before pushing up (and forgot this).

If you have IDs 1-100 and ordering ID asc, you end up saying `select * from employees where id < 100 limit 1` which gives you `1` instead of `99`. The linked solution is in the PR (SQL query the opposite order then reverse in-memory) but that was a little more than I had time for at the moment - and I think it's an open question if this is "worth it" with the extra complexity, or if we should only implement offset-based for now - so pulled it out.